autogen/cache/cache.py

```diff
@@ -26,7 +31,14 @@ class Cache(AbstractCache):
         cache: The cache instance created based on the provided configuration.
     """
 
-    ALLOWED_CONFIG_KEYS = ["cache_seed", "redis_url", "cache_path_root"]
+    ALLOWED_CONFIG_KEYS = [
+        "cache_seed",
+        "redis_url",
+        "cache_path_root",
+        "connection_string",
+        "database_id",
+        "container_id",
```

Comment:
How about merging all the three keys into one `cosmos_db_config`? That's more intuitive and easier to understand.